www/adminops

```diff
@@ -1289,12 +1289,12 @@ else if (isset($_REQUEST['users'])) {
         if ($rebuild || $sortTitle == "")
             $setVars[] = "sort_title = '"
                          . mysql_real_escape_string(strtoupper(
-                             getSortingTitle($title))) . "'";
+                             getSortingTitle($title)), $db) . "'";

```

Comment:
I tested this feature locally by going to http://localhost:8080/adminops and clicking "Rebuild ALL author/title sort keys in GAMES table" http://localhost:8080/adminops?fixsortkeys&rebuild

In main, it didn't work, in the branch, it did. (But I have no idea what it does, so… 🤷‍♂️)